source/options-storage.ts

```diff
@@ -31,8 +31,8 @@ for (const feature of __featuresList__) {
 }
 
 const migrations = [
-	featureWasRenamed('make-discussion-sidebar-sticky', 'sticky-discussion-sidebar'), // Merged on August 1st
 	featureWasRenamed('linkify-code', 'linkify-urls-in-code'), // Merged on September 1st
+	featureWasRenamed('highlight-collaborators-in-lists', 'highlight-collaborators-and-own-discussions'), // Merged on September 20th
```

Comment:
And `highlight-own-issues-and-prs` ?

Reply:
It will just be dropped by `removeUnused`

Reply:
Ah yes, so `highlight-collaborators-in-lists` will be leading.

Reply:
Yes. Migration are just to preserve disabled features, and `highlight-own-issues-and-prs` is such a minor feature to disable it just does not matter.